kcidb/test_monitor.py

```diff
@@ -198,5 +198,15 @@ def test_min(self):
             self.assertEqual(message['X-KCIDB-Notification-Message-ID'],
                              obj_type_name)
             self.assertIn(f"Test {obj_type_name}: ", message['Subject'])
+            text, html = message.get_payload()
+            self.assertEqual('text/plain', text.get_content_type())
+            self.assertEqual('utf-8', text.get_content_charset())
+            content = text.get_payload()
             self.assertIn(f"Test {obj_type_name} detected!\n\n",
-                          message.get_payload())
+                          content)
+            self.assertEqual('text/html', html.get_content_type())
+            self.assertEqual('utf-8', html.get_content_charset())
+            content = html.get_payload()
+            self.assertIn(f"Test {obj_type_name} detected!\n\n",
+                          content)
+                          
```

Comment:
Could you remove the spaces from this line?

Reply:
@spbnick I am not sure which line you are referring to. Is it 209 or 210?

Reply:
The last line in the file.

Reply:
Oh, okay.. Kindly review the change.

Reply:
Thanks, it's perfect now :)